core/src/main/resources/error/error-classes.json

```
@@ -1526,8 +1526,20 @@
   },
   "WRONG_NUM_ARGS" : {
     "message" : [
-      "The <functionName> requires <expectedNum> parameters but the actual number is <actualNum>."
-    ]
+      "Invalid number of arguments for the function <functionName>."
+    ],
+    "subClass" : {
+      "WITHOUT_SUGGESTION" : {
+        "message" : [
+          "Please ref to 'https://spark.apache.org/docs/latest/sql-ref-functions.html' for fix."
+        ]
+      },
+      "WITH_SUGGESTION" : {
+        "message" : [
+          "The function requires <expectedNum> parameters but the actual number is <actualNum>."
```

The sentence still looks like a description of an issue but not a suggestion.
```suggestion
          "Consider to change the number of arguments because the function requires <expectedNum> parameters but the actual number is <actualNum>."
```